bip-0360.mediawiki

```diff
@@ -1,724 +1,452 @@
 <pre>
-  BIP: 360
-  Layer: Consensus (soft fork)
-  Title: Pay to Quantum Resistant Hash
-  Author: Hunter Beast <hunter@surmount.systems>
-          Ethan Heilman <ethan.r.heilman@gmail.com>
-  Comments-Summary: No comments yet.
-  Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-0360
-  Status: Draft
-  Type: Standards Track
-  Created: 2024-12-18
-  License: BSD-3-Clause
+BIP: 360
+Title: Pay-to-Tapscript-Hash (P2TSH)
+Layer: Consensus (soft fork)
+Author: Hunter Beast <hunter@surmount.systems>
+        Ethan Heilman <ethan.r.heilman@gmail.com>
+Comments-Summary: No comments yet.
+Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-0360
+Status: Draft
+Type: Standards Track
+Created: 2024-12-18
 </pre>
 
-== Introduction ==
+'''Abstract'''
 
-=== Abstract ===
+This document proposes a new output type, Pay-to-Tapscript-Hash (P2TSH), via a soft fork. P2TSH outputs operate with nearly the same functionality as P2TR (Pay-to-Taproot) addresses – but with the key-path spend removed.
 
-This document proposes the introduction of a new output type, Pay to Quantum Resistant Hash (P2QRH), via a soft fork.
-P2QRH provides the same tapscript functionality as Pay to Taproot (P2TR) but removes the quantum-vulnerable
-key path spend in P2TR. By itself, P2QRH provides protection against long-exposure quantum attacks, 
-but requires PQ signatures to provide full security against Cryptographically Relevant Quantum Computers (CRQCs).
-P2QRH is designed to provide the foundation necessary for a future soft fork activating PQ signature verification 
-in tapscript.
+Through this modification, P2TSH addresses create the option of using Taproot in a manner that is:
 
-=== Copyright ===
+# resistant to potential attacks by cryptographically-relevant quantum computers (CRQCs), and
+# resistant to any future cryptanalytic approaches that could compromise elliptic curve cryptography (ECC) used by Bitcoin.
 
-This document is licensed under the 3-clause BSD license.
+It is worth noting that the proposed P2TSH outputs are only resistant to (so-called) “long-exposure attacks” on elliptic curve cryptography (ECC); that is, attacks on public keys exposed for time periods longer than the average time needed to confirm a bitcoin transaction.
```

Comment:
```suggestion
It is worth noting that the proposed P2TSH outputs are only resistant to “long-exposure attacks” on elliptic curve cryptography (ECC); that is, attacks on public keys exposed for time periods longer than the average time needed to confirm a bitcoin transaction.
```

Given that we are introducing the terminology "long-exposure" I'd drop (so-called), but if you really want it, keep it.

Reply:
We're on the same page here, its a novel definition.